charts/ovn/values.yaml

```diff
@@ -37,16 +38,16 @@ images:
 
 labels:
   ovn_ovsdb_nb:
-    node_selector_key: openstack-compute-node
+    node_selector_key: openstack-network-node
     node_selector_value: enabled
   ovn_ovsdb_sb:
-    node_selector_key: openstack-compute-node
+    node_selector_key: openstack-network-node
     node_selector_value: enabled
   ovn_northd:
-    node_selector_key: openstack-compute-node
+    node_selector_key: openstack-network-node
     node_selector_value: enabled
   ovn_controller:
-    node_selector_key: openstack-compute-node
+    node_selector_key: openvswitch
     node_selector_value: enabled
   ovn_controller_gw:
     node_selector_key: openstack-control-plane
```

Comment:
This will break existing deployments I believe.  We don't have `openstack-network-node` label on these systems.   Also, technically, ovsdb is just db and northd is just a control plane service, so i hink they should live on the control plane

also, this will now run ovn gw and normal ovn on control plane because both ds match, so we should keep labels as is.

Reply:
it should not affect us. I set them in ovn roles variable because upstream doesn't accept this.